packages/server/lib/controllers/v1/orb/postWebhooks.ts

```diff
@@ -29,7 +31,7 @@ export const postOrbWebhooks = asyncWrapper<PostOrbWebhooks>(async (req, res) =>
     const handled = await handleWebhook(req.body as Webhooks);
     if (handled.isErr()) {
         report(handled.error, { body: req.body });
-        res.status(500).send({ success: false });
+        res.status(500).send({ error: { code: 'server_error', message: handled.error.message } });
```

Comment:
Do we do anything about idempotency? I see Orb only guarantees _at least once delivery_  - would there be any undesirable side-effects if we receive the webhook twice?

I think this would be a candidate to move to pubsub when it's ready so that we do as little processing as possible in the webhook handler.